pkg/commands/image/create_test.go

```diff
@@ -901,10 +901,8 @@ status: {}
 			})
 		})
 
-		when("cache size is not provided", func() {
-			it("does not set an empty field on the image", func() {
-				//note: this is to allow for defaults to be set by kpack webhook
-
+		when("local-source and local—path-destination-image is provided", func() {
+			it("sets the repository source path on the image", func() {
```

Comment:
Let's move this entire `it` block to the "local source code" grouping on line 311

https://github.com/buildpacks-community/kpack-cli/blob/0c9899ac07316194fe605a871cceff8ad23fe2ae/pkg/commands/image/create_test.go#L311